packages/netdog/netdog.spec

```
@@ -53,21 +51,46 @@ Summary: Reverse DNS Hostname detector
 
 %package -n %{_cross_os}hostname-imds
 Summary: IMDS Hostname detector
+Requires: %{_cross_os}hostname-imds(binaries)
 %description -n %{_cross_os}hostname-imds
 %{summary}
 
+%package -n %{_cross_os}hostname-imds-bin
+Summary: IMDS Hostname detector binaries
+Provides: %{_cross_os}hostname-imds(binaries)
+Requires: (%{_cross_os}image-feature(no-fips) and %{_cross_os}hostname-imds)
+Conflicts: (%{_cross_os}image-feature(fips) or %{_cross_os}hostname-imds-fips-bin)
+%description -n %{_cross_os}hostname-imds-bin
+%{summary}
+
+%package -n %{_cross_os}hostname-imds-fips-bin
+Summary: IMDS Hostname detector binaries, FIPS edition
+Provides: %{_cross_os}hostname-imds(binaries)
+Requires: %{_cross_os}hostname-imds(binaries)
```

Copy/paste error; this shouldn't require the same capability that it provides:
```suggestion
Requires: (%{_cross_os}image-feature(fips) and %{_cross_os}hostname-imds)
```